### DISCUSSION TITLE
Console Schedule for range of specified seconds?

### DISCUSSION THREAD

**Discussion by nimmneun** [laravel/framework#51683] [Ideas]
Hey =)

Are there any plans to add something along the lines of ...

```php
ManagesFrequencies::onlySeconds([3, 5, 7, 33, 35, 37]);
```
... to https://github.com/laravel/framework/blob/11.x/src/Illuminate/Console/Scheduling/ManagesFrequencies.php#L72
to run a command only at the specified seconds?

I'm currently migrating a rather scheduling heavy project to Laravel and felt that it would be nicer instead of doing e.g. everySecond() and then checking `in_array(time() % 60, [3, 5, 7, 33, 35, 37])` or a for loop in combo with everyMinute().

Not a big deal ofc, I just thought it would be nice to have that option.

Have a great day =)

**Comment by henzeb** [comment]
You could just do everyFiveSeconds or similar.

If Schedule is Macroable, you can add the function yourself using ->at(value(...))

**Reply by nimmneun** [reply]
Thanks for the heads up regarding Macroable =)
I already solved it for myself by extending the default Schedule.
I just though it would would be a nice extra to have in general, since it's basically in line with cron's `every n/*` and `only n,...` pattern and in my case I need the run times to be on point and they are not evenly devisible.
